src/replication_test.c

```diff
@@ -287,6 +291,8 @@ send_mgmt_ack(int fd, zvol_op_code_t opcode, void *buf, char *replica_ip,
 		iovec_count = 2;
 	} else {
 		build_mgmt_ack_data;
+		if(delay > 0)
```

Comment:
are we introducing delays in mgmt connection as well? what is the testcase that we are doing with this?
and, why doing sleep for 5 seconds? why don't we use the value in 'delay' variable?

Reply:
Yes Sir, Intention is to kill non-quorum replica during mgmt connection. sleep for 5 sec is the minimum time required to error out the replica by killing. Any way I can even go with delay time. According to the test case, a small delay is enough so went with sleep for 5 sec.

Reply:
You may need to better use different flag for introducing 'delay' in mgmt requests. Now, every mgmt request will be delayed with existing flag.